sdk/dotnet/Cognito/IdentityPoolRoleAttachment.cs

```diff
@@ -30,7 +30,7 @@ public partial class IdentityPoolRoleAttachment : Pulumi.CustomResource
         /// The map of roles associated with this pool. For a given role, the key will be either "authenticated" or "unauthenticated" and the value will be the Role ARN.
         /// </summary>
         [Output("roles")]
-        public Output<Outputs.IdentityPoolRoleAttachmentRoles> Roles { get; private set; } = null!;
+        public Output<ImmutableDictionary<string, string>> Roles { get; private set; } = null!;
```

Comment:
Hmm, that's a breaking one. What changed upstream?

Reply:
https://github.com/terraform-providers/terraform-provider-aws/commit/877b365779abcc5d5b4c3e110999003d3fe7c096

Changed from a list to a map - this will stop the continual diffs

Reply:
It actually changed from a strongly-typed map to a weakly typed map in upstream, which leads to it changing from a class to a dictionary in .NET.  Thus it was not a breaking change in upstream but is a breaking change in .NET (and Go, but I believe not in TypeScript or Python).  

Let's add a note to the CHANGELOG about this change - perhaps:

> As part of updating to `v2.61.0`, the following properties have changed from being strongly typed classes to being loosely typed maps.  This may require changes to Go and .NET code using these properties:  `aws.cognito.IdentityPoolRoleAttachment#roles` and `aws.ssm.Document#permissions`.